src/Tools/RepoUtil/ConflictingPackagesException.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RepoUtil
+{
+    internal class ConflictingPackagesException : Exception
+    {
+        public ConflictingPackagesException(List<NuGetPackageConflict> conflictingPackages) :
+            base("Creation failed because of conflicting package versions.")
+        {
+            ConflictingPackages = conflictingPackages;
```

Comment:
I'd change `ConflictingPackages` to be an `ImmutableArray<NuGetPackageConflicT>` items.  In general exception data should be immutable.